code/src/index.css

```diff
@@ -1,13 +1,183 @@
+* {
+	margin: 0;
+	padding: 0;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 20px;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	background-color: #181818;
+	cursor: default;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+	font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New", monospace;
+}
+
+header {
+	width: 300px;
+	margin: 0 auto;
+}
+
+.heading {
+	font-family: Helvetica;
+	font-size: 18px;
+	color: #ffffff;
+}
+
+hr {
+	opacity: 0.1;
+}
+
+.album-container {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	column-gap: 20px;
+	row-gap: 45px;
+	padding-top: 20px;
+}
+
+.album-image-container {
+	position: relative;
+}
+
+.album-image {
+	width: 300px;
+	display: block;
+	margin-bottom: 10px;
+}
+
+.album-cover-overlay {
+	opacity: 0;
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	padding: 112.5px 42.5px;
+	background-color: rgba(0, 0, 0, 0.5);
+}
+
+.album-cover-overlay:hover {
+	opacity: 1;
+}
+
+.icons-container {
+	display: flex;
+	justify-content: center;
+	gap: 40px;
+	position: relative;
+}
+
+img.heart-icon {
+	width: 30px;
+}
+
+img.play-icon {
+	width: 75px;
+	transition: all 0.2s ease-in-out;
+}
+
+img.play-icon:hover {
+	transform: scale(1.05);
+}
+
+img.dots-icon {
+	width: 30px;
+}
+
+img.icon {
+	opacity: 0.6;
+}
+
+img.icon:hover {
+	cursor: pointer;
+	opacity: 1;
+}
+
+a {
+	text-decoration: none;
+}
+
+.album-artist-title-container {
+	width: 300px;
+}
+
+.artists-container {
+	margin-top: 6px;
+}
+
+.album-title {
+	font-family: Helvetica;
+	font-size: 14px;
+	color: #ffffff;
+	font-weight: bold;
+}
+
+.album-artist {
+	font-family: Helvetica;
+	font-size: 14px;
+	color: #a0a0a0;
+	font-weight: bold;
+}
+
+.album-title:hover,
+.album-artist:hover {
+	text-decoration: underline;
+}
+
+footer {
+	padding-top: 70px;
+}
+
+.creators {
+	font-family: Helvetica;
+	font-size: 14px;
+	color: #a0a0a0;
+	margin-top: 6px;
+	text-align: center;
+}
+
+@media (min-width: 660px) {
+	header {
+		width: 660px;
+	}
+}
+
+@media (min-width: 980px) {
+	header {
+		width: 980px;
+	}
+}
+
+@media (min-width: 1300px) {
+	header {
+		width: 1300px;
+	}
+}
+
+@media (min-width: 1620px) {
+	header {
+		width: 1620px;
+	}
+}
+
+@media (min-width: 1778px) {
+	header {
+		width: 1680px;
+	}
+}
+
+@media (min-width: 1903px) {
+	header {
+		width: 1770px;
+	}
+}
+
+@media (min-width: 1940px) {
+	header {
+		width: 1900px;
+	}
```

Comment:
For the future instead of adding this amount of media queries you could have benefited by using a percentage 
 on margin-left or the entire width of the header for the header text. It would then follow seamlessly and stay inline with the top left album.